libkbfs/crypto_common.go

```diff
@@ -294,15 +294,15 @@ func (c CryptoCommon) DecryptPrivateMetadata(
 
 const minBlockSize = 256
 
-// nextPowerOfTwo returns next power of 2 greater than the input n.
+// thisPowerOfTwo returns next power of 2 greater than or equal to the input n.
 // https://en.wikipedia.org/wiki/Power_of_two#Algorithm_to_round_up_to_power_of_two
-func nextPowerOfTwo(n uint32) uint32 {
-	if n < minBlockSize {
+func thisPowerOfTwo(n int) int {
```

Comment:
Ok, I guess I'm fine with this, then.  But @songgao I find the name a little confusing, maybe just `powerOfTwo`, or `powerOfTwoEqualOrGreater`.  Also, looks like the wikipedia link has changed slightly: https://en.wikipedia.org/wiki/Power_of_two#Algorithm_to_find_a_power_of_two_greater_than_or_equal_to_a_number

It seem like we might not need the `if` case to check if it's already a power of two.